code/workspaces/infrastructure-api/resources/datalab-dask-worker.deployment.template.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{name}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      dask-infra: {{workerPodLabel}}
+  template:
+    metadata:
+      labels:
+        dask-infra: {{workerPodLabel}}
+    spec:
+      containers:
+        - image: {{{daskImage}}}
```

Comment:
Is there an extra set of `{ }` here?

Reply:
Yes - this prevents mustache HTML-escaping, which we need for the daskImage because it has slashes and colons.  See https://www.npmjs.com/package/mustache.

Reply:
Ahhh I see! I have only seen the escaping using an `&` before